src/Compilers/CSharp/Portable/Errors/.editorconfig

```diff
@@ -0,0 +1,4 @@
+# CSharp code style settings:
+[*.cs]
+# CA1069: Enums should not have duplicate values
+dotnet_diagnostic.CA1069.severity = warning
```

Comment:
💡 These lines should appear in .globalconfig instead of .editorconfig for performance and consistent application

Reply:
The compiler team doesn't currently desire consistent application of this warning. Is there a change that we should make here to improve performance which doesn't affect which files are subject to the new warning?

Reply:
For intentional conditional application, .editorconfig is fine.